tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsManager.cs

```diff
@@ -53,16 +53,7 @@ internal class DataStreamsManager
         IDiscoveryService discoveryService,
         string defaultServiceName)
     {
-        var enableDsm = settings.IsDataStreamsMonitoringEnabled;
```

Comment:
Talk about early retirement, this code was in place for what, 3 commits 😆

Reply:
The important thing is that it's in the release 😉